plugin/src/Main.server.lua

```diff
@@ -55,15 +55,21 @@ local function main()
 				end)
 		end)
 
-	toolbar:CreateButton("Sync In", "Sync into Roblox Studio", Config.icons.syncIn)
-		.Click:Connect(function()
-			checkUpgrade()
+	local function syncIn()
+		checkUpgrade()
 
-			pluginInstance:syncIn()
-				:catch(function(err)
-					warn(err)
-				end)
-		end)
+		pluginInstance:syncIn()
+			:catch(function(err)
+				warn(err)
+			end)
+	end
+
+	local dShort = "Sync In"
+	local dLong = "Sync into Roblox Studio"
```

Comment:
Can we spell out these variable names? `shortLabel` and `longLabel` would be fine I imagine.